packages/new-deploy/src/DeployContext.ts

```diff
@@ -280,6 +291,9 @@ export default class DeployContext extends MultiProvider<config.Domain> {
 
     await this.relinquish();
 
+    await this.checkCores();
```

Comment:
improvement idea for another PR:
I feel like there should be checks before `relinquish` so that we error into a recoverable state